lib/components/src/blocks/Story.tsx

```diff
@@ -98,11 +98,14 @@ const IFrameStory: React.FunctionComponent<IFrameStoryProps> = ({
 const Story: React.FunctionComponent<StoryProps> = props => {
   const { error } = props as ErrorProps;
   const { storyFn } = props as InlineStoryProps;
-  const { id, inline, title, height } = props;
+  const { id, inline, title, height, children } = props;
 
   if (error) {
     return <EmptyBlock>{error}</EmptyBlock>;
   }
+  if (!children) {
```

Comment:
I think this `Story` never gets any children passed. If you have access to the Chromatic build (linked in the Github status checks) you can see how it breaks some of the examples.

`addon-docs/blocks/Story` parses out the `storyFn` and that gets passed in. So I think the check should be done inside `InlineStory` & `IFrameStory` respectively.

Reply:
I might be missing something - I don't see this (but I dont really know the chromatic build). 

I added the fix there (show a "No component or story to display'" message) because I saw also the other error check is being in there. 
```
if (error) {
    return <EmptyBlock>{error}</EmptyBlock>;
  }
```

Also, on my end, it breaks as soon as I add the sample Story from the issue, doesn't to break for you too even outside the chromatic build. And child are passed as null in that case. 
```
<Preview>
  <Story id="nonexistent-story" />
</Preview>
```

Reply:
I think something is wrong with your setup. Running this on my local machine matches with Chromatic is showing. Try running:

```
yarn bootstrap --reset --core
```

This will clear out your `node_modules` and reinstall from scratch. It will also delete ALL files in the monorepo that are not in git, so **BE CAREFUL** with this command.

Reply:
you are right that my config did not match yours, here is an update it might help
1. tried to rebuild gave me an error building node_modules/weak
`error /Users/atanasster/storybook/node_modules/weak: Command failed.` from:
`../src/weakref.cc:147:25: error: too few arguments to function call, expected 4, have 3` etc.
2. turns out I had `node --version: v12.1.0`
3. switched down to `node --version v10.16.3`
4. `yarn bootstrap --reset --core` - the build worked
 
No idea how/if the original build worked since I've had node 12 for several months now, maybe building storybook was so new I didn't really pay attention if some parts of the build failed

Now I see the same situation as you do, looks like the issue was introduced with the fix for react functional component stories with hooks, now invalid stories have the issue rendering i will update this PR as soon as I figure out what's happening, just wanted to give a heads up that it seems the build has issues under node 12.xx.

Reply:
Hmm maybe somehow `yarn dev` succeeded where `yarn bootstrap` failed? `bootstrap` does more work than `dev`.

Reply:
aaaah, I think you are right - I probably never did a full build